src/Footer.js

```diff
@@ -1,58 +1,58 @@
 import React from 'react'
 import "./footer.css"
-import {useState} from 'react';
+import { useState } from 'react';
 import Divider from '@material-ui/core/Divider';
 function Footer() {
 
 
-//back to top-->
-const [visible, setVisible] = useState(false)
+  //back to top-->
+  const [visible, setVisible] = useState(false)
   const toggleVisible = () => {
     const scrolled = document.documentElement.scrollTop;
-    if (scrolled > 300){
+    if (scrolled > 300) {
       setVisible(true)
-    } 
-    else if (scrolled <= 300){
+    }
+    else if (scrolled <= 300) {
       setVisible(false)
     }
   };
-  const scrollToTop = () =>{
+  const scrollToTop = () => {
     window.scrollTo({
-      top: 0, 
+      top: 0,
       behavior: 'smooth'
     });
   };
   window.addEventListener('scroll', toggleVisible);
 
-    return (
-      <footer className="footer">
-        <div className="top" onClick={scrollToTop}>
-              Back to top
+  return (
```

Comment:
@void-ness , Could you please add a screenshot of the changes you have done in the footer?

Reply:
Original | Updated
 :--------------------: |:--------------------:
![image](https://user-images.githubusercontent.com/78547746/194640697-c634d51d-f239-4caa-9df4-7256eac3d7db.png) | ![image](https://user-images.githubusercontent.com/78547746/194639855-c199bfe9-a649-4423-8456-30c71ea91a1d.png)

Earlier the image was not loading due to incorrect value in src attribute for the image.
https://github.com/Kanika637/amazon-clone/blob/6a789738ecdb8a35437753b3017e8ca8a5c017d1/src/Footer.js#L127-L129

Here's the updated code snippet
![image](https://user-images.githubusercontent.com/78547746/194641200-c9fab7ce-e5b7-408d-9806-0014d3f898da.png)

I have a prettier extension installed which might have increased the number of changes in the given file 😅